e2e/netman/pingpong/pingpong.go

```diff
@@ -123,7 +123,7 @@ func (d *XDapp) fund(ctx context.Context) error {
 			return err
 		}
 
-		fund := new(big.Int).Div(big.NewInt(params.Ether), big.NewInt(10)) // Fund it with 0.1 ETH
+		fund := new(big.Int).Div(big.NewInt(params.Ether), big.NewInt(2)) // Fund it with 0.5 ETH
```

Comment:
Note I needed to bump ping pong funding, because we are charging real fees now. 

Could use some more visibility into actual fees paid - added a [ticket](https://app.asana.com/0/1206208509925075/1207199096986774) to add fees to XMsg events.